### ISSUE TITLE
Release this on the world?

### ISSUE THREAD

**Issue by mklingen** [personalrobotics/or_rviz#12]
What do you think about putting a post on OpenRAVE users about this? I think people would find this useful.

**Comment by siddhss5** [comment]
Sounds like a great idea once @mkoval and @psigen agree.
It'd also be nice to have a YouTube video that showcases the viewer. People like videos :).

**Comment by psigen** [comment]
I'm strongly in support, it's a very useful plugin... but first we need _documentation._

Happy to help write that up.

**Comment by mklingen** [comment]
Yep. Very happy and willing to write documentation and make videos whenever we want to do this.

**Comment by mklingen** [comment]
The documentation is looking better and better now. What's left before we can announce this? (btw we already have an invention disclosure.)

**Comment by mkoval** [comment]
I'm going to go ahead and rename this repository/package to or_rviz, like we discussed. At that point I think we're ready to release it.

**Comment by mkoval** [comment]
I just did the rename in #17. I also imported `master` of the old or_rviz repository as the branch `backup/or_rviz` and renamed the old repository to or_rviz_backup.

@mklingen Can you take a look? Assuming that contains everything we care about, I'll delete the backup repository.

**Comment by siddhss5** [comment]
It'd be great if Matt could make a video that showcases the features of the visualizer and put it on our lab's YouTube channel.

**Comment by mklingen** [comment]
@siddhss5 will do.

**Comment by mklingen** [comment]
I made a couple of youtube videos:

https://www.youtube.com/watch?v=ZniUFJZ5kHg

https://www.youtube.com/watch?v=vzcm8KG3S_U

**Comment by cdellin** [comment]
Those look awesome!  Maybe throw them together and slap a title slide á la the or_ompl demo video [0] and be done with it?

[0] https://www.youtube.com/watch?v=6qRRbvNzHG8

**Comment by mkoval** [comment]
Those videos look great! I agree with @cdellin: it would be great if you could combine the two videos and add title slides or overlays to explain what's going on.

**Comment by mklingen** [comment]
I made an integrated video. It will be up here soon: https://www.youtube.com/watch?v=3trBTwgC4fo&feature=youtu.be

There is a bit of minor frame skipping because of the .ogv to .avi conversion messing up.

**Comment by siddhss5** [comment]
There's a massive drop in resolution in this video and it looks very
blocky. Can you fix that?
Otherwise, looks good.
Sidd.

**Comment by mklingen** [comment]
Sidd, I think that Youtube just hasn't finished processing it in HD yet. It only displays at 480p.

**Comment by mklingen** [comment]
Nvm, I messed up and uploaded a 480p video. Re-rendering.

**Comment by mklingen** [comment]
re-uploaded a 720p video. Will be live here when done: https://www.youtube.com/watch?v=DUA3jc7CAoI&feature=youtu.be

**Comment by mkoval** [comment]
That looks great! I think we should make an announcement on openrave-users.

@siddhss5 Is that okay with you?

**Comment by siddhss5** [comment]
I'd like to see the final video before doing that.

**Comment by mkoval** [comment]
@siddhss5 Did you see @mklingen's second video? I don't see any resolution issues in that one.

**Comment by siddhss5** [comment]
Aha, awesome, let's go with it. @mkoval, could you please type it out and sent it to the gang for confirmation before emailing it to `openrave-users`?

**Comment by mkoval** [comment]
I just sent you a draft of the announcement via email. Let me know what you think.

@mklingen The YouTube video has the incorrect URL`http://github.personalrobotics/or_rviz` on the title slide. Could you change this to `https://github.com/personalrobotics/or_rviz`?

**Comment by mklingen** [comment]
@mkoval  I covered up my mistake with an ugly annotation.

**Comment by mkoval** [comment]
Done:

http://openrave-users-list.185357.n3.nabble.com/or-rviz-OpenRAVE-viewer-plugin-built-on-RViz-td4027210.html
